TCAT/WhatsNewHeaderView.swift

```diff
@@ -0,0 +1,155 @@
+        
+        backgroundView.addSubview(whatsNewHeader)
+    }
+    
+    func createUpdateTitle() {
+        updateTitle = UILabel()
+        updateTitle.text = "App Shortcuts for Favorites"
+        updateTitle.font = UIFont(name: Constants.Fonts.SanFrancisco.Bold, size: 16)
+        
+        backgroundView.addSubview(updateTitle)
+    }
+    
+    
+    func createUpdateDescription () {
+        updateDescription = UILabel()
+        updateDescription.text = "Force Touch the app icon to search your favorites even faster."
+        updateDescription.font = UIFont(name: Constants.Fonts.SanFrancisco.Regular, size: 14)
+        updateDescription.textColor = UIColor(red: 144/255, green: 148/255, blue: 156/255, alpha: 1)
+        updateDescription.numberOfLines = 0
+        updateDescription.textAlignment = .center
+        
+        backgroundView.addSubview(updateDescription)
+    }
+    
+    func createDismissButton() {
+        dismissButton = UIButton()
+        dismissButton.setTitle("OK", for: .normal)
+        dismissButton.titleLabel?.font = UIFont(name: Constants.Fonts.SanFrancisco.Semibold, size: 14)
+        dismissButton.addTarget(self, action: #selector(okButtonPressed), for: .touchUpInside)
+        dismissButton.backgroundColor = UIColor(red: 8/255, green: 160/255, blue: 224/255, alpha: 1)
+        dismissButton.setTitleColor(.white, for: .normal)
+        dismissButton.layer.cornerRadius = 14.5
+        dismissButton.clipsToBounds = true
+        
+        backgroundView.addSubview(dismissButton)
+    }
+    
+    override func updateConstraints() {
+        super.updateConstraints()
+        backgroundView.snp.makeConstraints { (make) in
+            make.centerX.equalToSuperview()
+            make.width.equalTo(UIScreen.main.bounds.width - 32)
+            make.height.equalTo(150)
+            make.bottom.equalTo(snp.bottom)
+        }
+        
+        whatsNewHeader.snp.makeConstraints { (make) in
+            make.centerX.equalToSuperview()
+            make.height.equalTo(whatsNewHeader.intrinsicContentSize.height)
+            make.width.equalTo(whatsNewHeader.intrinsicContentSize.width)
+            make.bottom.equalTo(updateTitle.snp.top).offset(-8)
+        }
+        
+        updateTitle.snp.makeConstraints { (make) in
+            make.centerX.equalToSuperview()
+            make.height.equalTo(updateTitle.intrinsicContentSize.height)
+            make.width.equalTo(updateTitle.intrinsicContentSize.width)
+            make.bottom.equalTo(updateDescription.snp.top).offset(-6)
+        }
+        
+        updateDescription.snp.makeConstraints { (make) in
+            make.bottom.equalTo(dismissButton.snp.top).offset(-12)
+            make.centerX.equalToSuperview()
+            if let text = updateDescription.text {
+                make.height.equalTo(text.heightWithConstrainedWidth(width: 300, font: updateDescription.font))
```

Comment:
Why did you choose to do this instead of adding a padding of 20? Always be wary of big numbers for constraints.